src/plugins/ovp/services/JwplayerMediaSubscriber.js

```diff
@@ -0,0 +1,71 @@
+import EventSubscriber from '@gdbots/pbjx/EventSubscriber';
+import getTextFieldError from '@triniti/cms/components/text-field/getTextFieldError';
+import Microtime from '@gdbots/pbj/well-known/Microtime';
+
+export default class JwplayerMediaSubscriber extends EventSubscriber {
+  constructor() {
+    super();
+    this.onInitForm = this.onInitForm.bind(this);
+    this.onValidateForm = this.onValidateForm.bind(this);
+    this.onSubmitForm = this.onSubmitForm.bind(this);
+  }
+
+  /**
+   * Runs when the form is first created and is used to
+   * update initial values.
+   *
+   * @param {FormEvent} formEvent
+   */
+  onInitForm(formEvent) {
+    const data = formEvent.getData();
+    const node = formEvent.getMessage();
+
+    data.jwplayerMediaId = node.get('jwplayer_media_id');
+    data.jwplayerSyncEnabled = node.get('jwplayer_sync_enabled');
+    data.jwplayerSyncedAt = node.has('jwplayer_synced_at')
+      ? Microtime.fromString(String(`${node.get('jwplayer_synced_at', 1574813280)}000000`)).toMoment().format('MMM DD, YYYY hh:mm A')
```

Comment:
@efjacobson remove that default hard coded timestamp also you can just pass the timestamp directly to moment and avoid the creation of a microtime.

Reply:
@gdbrown done